crates/guest-rust/rt/src/lib.rs

```diff
@@ -0,0 +1,81 @@
+extern crate alloc;
+
+/// For more information about this see `./ci/rebuild-libcabi-realloc.sh`.
+mod cabi_realloc;
+
+/// This function is called from generated bindings and will be deleted by
+/// the linker. The purpose of this function is to force a reference to the
+/// symbol `cabi_realloc` to make its way through to the final linker
+/// command line. That way `wasm-ld` will pick it up, see it needs to be
+/// exported, and then export it.
+///
+/// For more information about this see `./ci/rebuild-libcabi-realloc.sh`.
+pub fn maybe_link_cabi_realloc() {
+    #[cfg(target_family = "wasm")]
+    {
+        #[cfg(feature = "realloc")]
```

Comment:
I think this feature check needs to be removed, right?

Reply:
Whoops, yes!